src/SFA.DAS.Commitments.Database/StoredProcedures/GetDataLockStatusExpiryCandidates.sql

```diff
@@ -0,0 +1,27 @@
+﻿CREATE PROCEDURE [dbo].[GetDataLockStatusExpiryCandidates]
+	@BeforeDate DATETIME ,
+	@ExpirableErrorCodes int
```

Comment:
I suspect that all data locks in the db are candidates, irrespective of error code. So we probably don't need this or the associated config, which is an opportunity to get it wrong. I can't imagine a datalock of any type, now or introduced in future, that would never expire.